post-processor/vagrant/libvirt.go

```diff
@@ -64,26 +63,23 @@ func (p *LibVirtProvider) KeepInputArtifact() bool {
 	return false
 }
 func (p *LibVirtProvider) Process(ui packersdk.Ui, artifact packersdk.Artifact, dir string) (vagrantfile string, metadata map[string]interface{}, err error) {
-	diskName := artifact.State("diskName").(string)
+	disks := []map[string]string{}
+	format := artifact.State("diskType").(string)
 
 	// Copy the disk image into the temporary directory (as box.img)
-	for _, path := range artifact.Files() {
-		if strings.HasSuffix(path, "/"+diskName) {
-			ui.Message(fmt.Sprintf("Copying from artifact: %s", path))
-			dstPath := filepath.Join(dir, "box.img")
-			if err = CopyContents(dstPath, path); err != nil {
-				return
-			}
+	for i, path := range artifact.Files() {
+		ui.Message(fmt.Sprintf("Copying from artifact: %s", path))
+		diskName := fmt.Sprintf("box_%d.img", i)
+		dstPath := filepath.Join(dir, diskName)
+		disks = append(disks, map[string]string{
+			"path":   diskName,
+			"format": format,
```

Comment:
Is there a reason we don't include the `size` for each disk? It seems to be supported by the provider, but I'm not sure it is actually required.

Reply:
The size of disks is recovered by vagrant-libivirt here: https://github.com/vagrant-libvirt/vagrant-libvirt/blob/main/lib/vagrant-libvirt/action/handle_box_image.rb#L44
So, we don't need specify the size in json.